react/features/participants-pane/components/web/ParticipantsCounter.tsx

```diff
@@ -9,15 +9,18 @@ const useStyles = makeStyles()(theme => {
     return {
         badge: {
             backgroundColor: theme.palette.ui03,
-            borderRadius: '100%',
+            borderRadius: '12px',
             height: '16px',
-            width: '16px',
+            minWidth: '16px',
             color: theme.palette.text01,
             ...withPixelLineHeight(theme.typography.labelBold),
+	    lineHeight: '20px',
```

Comment:
Please remove this. The line-height is part of the typography (set in the line above)